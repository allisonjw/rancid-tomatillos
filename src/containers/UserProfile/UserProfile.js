import React from 'react';
import '../App/App.scss';

const UserProfile = () => {
  return (
    <section className="userProfile-container">
      <h2 className="userProfile-container-greeting">Hello, Name!</h2>
      <button className="userProfile-container-btn" aria-label="user logout">Logout</button>
    </section>
  )
}

export default UserProfile;
