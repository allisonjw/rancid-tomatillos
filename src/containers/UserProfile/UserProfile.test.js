import React from 'react';
import { render } from '@testing-library/react';
import { shallow } from 'enzyme';
import UserProfile from './UserProfile';


describe('UserProfile', () => {
  let wrapper;

  beforeEach(() => {

    wrapper = shallow(
        <UserProfile
        />
    )

  })

  it('should render with correct data', () => {
    expect(wrapper).toMatchSnapshot()
  })

})
