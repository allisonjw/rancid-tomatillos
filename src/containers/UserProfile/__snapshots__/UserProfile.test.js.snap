// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserProfile should render with correct data 1`] = `
<section
  className="userProfile-container"
>
  <h2
    className="userProfile-container-greeting"
  >
    Hello, Name!
  </h2>
  <button
    className="userProfile-container-button"
  />
</section>
`;
